### PR TITLE
Add features alert

### PR DESCRIPTION
**Pull request by alan-wu**
The changes in this pull request adds an alert message to the provenance UI if there is one.

image

image

### CODE REVIEW

File: src/components/FlatmapVuer.vue [hunk 1]
```diff
@@ -317,31 +319,42 @@
                   key="layersSelection"
                 />
               -->
+              <selections-group
+                v-if="pathways && pathways.length > 0"
+                title="Pathways"
+                labelKey="label"
```
Comment (Tehsurfer) [inline src/components/FlatmapVuer.vue:325]: image
Do we want Alert visibility be first thing in the visibility list?

File: src/components/FlatmapVuer.vue [hunk 2]
```diff
@@ -869,6 +880,99 @@ export default {
...
+          };
+        }
+      } else {
+        if (alert.without) {
+          filter = {
+            NOT: { HAS: 'alert'},
+          };
+        } else {
+          filter = {
+            AND: [ {NOT: { HAS: 'alert'}}, { HAS: 'alert'}]
```
Comment (Tehsurfer) [inline src/components/FlatmapVuer.vue:899]: these are mapboxgl filters right?

**Review by Tehsurfer — APPROVED** [review]
I would put the Alert vis selector at the bottom of the selector risk,
 
 Everything works well though :)

File: src/components/FlatmapVuer.vue (hunk 1 above)
Comment (alan-wu) [reply src/components/FlatmapVuer.vue:325]: I think the fCCB decided that it is important to make sure people are aware of the alert.

File: src/components/FlatmapVuer.vue (hunk 2 above)
Comment (alan-wu) [reply src/components/FlatmapVuer.vue:899]: Yes, they are. Let me know if there is a better way to do this one.

**Pull request merged**